server/authentication/oidc/oidc.go

```diff
@@ -22,6 +23,8 @@ type RemoteOidcAuthenticator struct {
 
 	JwksURI string
 	JWKs    *keyfunc.JWKS
+
+	httpClient retryablehttp.RetryableHTTPClient
```

Comment:
This should be `*http.Client` and we can inject it with

`retryClient.StandardClient()`